umbraco/Umbraco.CMS.Backoffice
# Feature: User avatar

This PR finalizes the user avatar upload feature.


How to test:
* Make sure you can upload an avatar for a user that is not you
* Make sure you can upload an avatar for your user. Check that the current user avatar in the top right corner updates.
* Make sure that the avatar in the user grid + user table is updated.
